src/utils/date.ts

```diff
@@ -145,6 +160,8 @@ function getWeekdayNames(locale: LocaleType) {
       return ['Dom', 'Seg', 'Ter', 'Qua', 'Qui', 'Sex', 'Sab'];
     case 'zh':
       return ['日', '一', '二', '三', '四', '五', '六'];
+    case 'ko':
+      return ['일', '월', '화', '수', '목', '금', '토'];
```

Comment:
Hi, are these translation correct? 
These are the translations I found, please correct me if I'm wrong
```
Monday -월요일
Tuesday - 화요일
Wednesday - 수요일
Thursday - 목요일
Friday - 금요일
​Saturday - 토요일
Sunday - 일요일
```

Reply:
Both are correct.
What I wrote is an abbreviation. 
I'm affraid there is little space. but it doesn't matter.
I will follow your judgement
Thanks for the quick response!